src/wkt/src/internal.rs

```diff
@@ -12,6 +12,26 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+pub struct F32;
```

Comment:
Oh, this is clever.... 

Optional: you may be able to remove the `float_serialize()` and `float_deserialize()` functions.

Reply:
Inlined!